README.md

```diff
@@ -149,6 +149,28 @@ in the `context["params"]` variable, e.g. getting a training data you would use
 
     For more [batch prediction settings](https://datarobot-public-api-client.readthedocs-hosted.com/en/v2.27.1/autodoc/api_reference.html#batch-predictions) see the DataRobot docs.
 
+- `GetTargetDriftOperator` - Gets the target drift from a deployment. Returns a JSON blob of the target drift data.
+
+    Parameters:
+
+        deployment_id: str - DataRobot deployment ID
+
+    Required config params:
+
+        None
+
+- `GetFeatureDriftOperator` - Gets the feature drift from a deployment. Returns a JSON blob of the feature drift data.
+
+    Parameters:
+
+        deployment_id: str - DataRobot deployment ID
+
+    Required config params:
+
+        None
+
+
```

Comment:
You may add [this link](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#datarobot.models.Deployment.get_feature_drift) to the docs for optional parameters.